src/app/EventManagement.h

```diff
@@ -419,6 +424,16 @@ class EventManagement
      */
     CHIP_ERROR EnsureSpaceInCircularBuffer(size_t aRequiredSpace);
 
+    /**
+     * @brief Iterate the event elements inside event tlv and mark the fabric index as 0 if matching with obsolete fabric index
+     *
+     * @param[in] aReader  event tlv reader
+     * @param[in] aDepth   the current depth into the TLV data.
```

Comment:
How about just not giving this arg a name, here or in the impl, since we don't use it?

Reply:
sounds good. updated, thanks